packages/client/rubygem-foreman_scap_client/rubygem-foreman_scap_client.spec

```diff
@@ -1,62 +1,39 @@
+# template: default
 %global gem_name foreman_scap_client
-%global config_dir %{_sysconfdir}/%{gem_name}
-
-%define rubyabi 1.8
 
 Name: rubygem-%{gem_name}
-Version: 0.5.0
+Version: 0.5.1
 Release: 1%{?dist}
-Summary: Client script that runs OpenSCAP scan and uploads the result to foreman proxy
-Group: Development/Languages
+Summary: Client script that runs openscap scan and uploads the result to foreman proxy
 License: GPLv3
-URL: https://github.com/openscap/foreman_scap_client
+URL: https://github.com/theforeman/foreman_scap_client
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
-Requires: bzip2
-Requires: ruby(rubygems)
-%if 0%{?el6}
-Requires: ruby(abi)
-Requires: rubygem(json) >= 1.4
-Requires: rubygem(json) < 2.0
-%else
-Requires: ruby(release)
-%endif
-Requires: openscap-scanner
-%if 0%{?el6}
-BuildRequires: ruby(abi)
-%else
-BuildRequires: ruby(release)
-%endif
-BuildRequires: ruby(rubygems)
-BuildRequires: rubygems-devel
+# start specfile generated dependencies
+Requires: ruby
 BuildRequires: ruby
+BuildRequires: rubygems-devel
 BuildArch: noarch
-Provides: rubygem(%{gem_name}) = %{version}
-
+# end specfile generated dependencies
 
 %description
-Client script that runs OpenSCAP scan and uploads the result to foreman proxy.
+Client script that runs openscap scan and uploads the result to foreman proxy.
 
 
 %package doc
 Summary: Documentation for %{name}
-Group: Documentation
 Requires: %{name} = %{version}-%{release}
 BuildArch: noarch
 
 %description doc
 Documentation for %{name}.
 
 %prep
-gem unpack %{SOURCE0}
-
-%setup -q -D -T -n  %{gem_name}-%{version}
-
-gem spec %{SOURCE0} -l --ruby > %{gem_name}.gemspec
+%setup -q -n  %{gem_name}-%{version}
```

Comment:
the template we use "expects" el8 or newer tooling, but this gem is also built for el7 (and el6 downstream? i don't know) and thus can't use the updated template.